businesscentral-monitoring/artifact-overrides.yaml

```diff
@@ -9,5 +9,4 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.6.GA-CR1-patch.zip"
-    md5: "2626e4536c4d7b2a0138ba1f50ecff58"
```

Comment:
As we are moving to 7.4.0 and patch is not needed - isn't needed change of start-up scripts to not patch Eap? Or when is patch not provided this step is skipped?

Reply:
This file was never used to apply EAP's patches.
probably was left there as example. :)
I honestly don't remember.